include/csoundCore.h

```diff
@@ -1072,24 +1265,48 @@ typedef struct _message_queue_t_ {
 
     /** @name Attributes */
     /**@{ */
-     /** Get number of output channels */
+    /** Get number of output channels */
     uint32_t (*GetNchnls)(CSOUND *);
     /** Get number of input channels */
     uint32_t (*GetNchnls_i)(CSOUND *);
     MYFLT (*Get0dBFS) (CSOUND *);
+
+    /** Get number of control blocks elapsed */
+    uint64_t (*GetKcounter)(CSOUND *);
+    MYFLT (*GetA4)(CSOUND *);
+    int (*GetTieFlag)(CSOUND *);
```

Comment:
Shouldn't this get the tie-status from the INSDS*? Seems like getting it from CSOUND* would interfere with parallel processing. (Same comment for GetReinitFlag)

Reply:
Noticing these functions were like this and re-organized in this PR, so the above shouldn't hold this PR back, but noting the oddity here.

Reply:
I think the GetKcounter should be removed as it is INSDS-based, not sure about 
the tie flag function though. The GetA4() is ok.